python/rmm/rmm.py

```diff
@@ -11,15 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import ctypes
-
-from cuda.cuda import CUdeviceptr, cuIpcGetMemHandle
-from numba import config, cuda
-from numba.cuda import HostOnlyCUDAMemoryManager, IpcHandle, MemoryPointer
-
-import rmm
-from rmm import _lib as librmm
-from rmm._cuda.stream import Stream
+from . import mr
```

Comment:
While I personally prefer relative imports. I think we use them fairly sparingly. I would like to use them more, but probably best to stay internally consistent until we can have that discussion?

Reply:
Done.